lib/WebService/Braintree/Configuration.pm

```diff
@@ -1,14 +1,96 @@
 package WebService::Braintree::Configuration;
 
+=head1 NAME
+
+WebService::Braintree::Configuration
+
+=head1 PURPOSE
+
+This keeps all configuration information for your WebService::Braintree
+installation.
+
+A singleton of this class is instantiated when you use L<WebService::Braintree>.
+You are intended to set attributes of this class immediately, then the rest of
+the distribution knows what to do.
+
+=cut
 
 use WebService::Braintree::Gateway;
 use Moose;
 
-has merchant_id => (is => 'rw');
+# IS THIS UNUSED? I cannot find reference in the current documentation for Ruby
```

Comment:
The partner_id is what I'm referring to. `grep -r lib partner_id` doesn't find it anywhere.